helm/prometheus-rules/templates/platform/atlas/alerting-rules/app-configuration.yml

```
@@ -0,0 +1,40 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: atlas-app-configuration.rules
+  namespace: {{ .Values.namespace }}
+spec:
+  groups:
+  - name: atlas-app-configuration
+    rules:
+    # Coming from https://gigantic.slack.com/archives/C07A03AN9JM
+    # This alert ensures our app has no unexpected configmaps.
+    - alert: ConfigmapUnexpected
+      annotations:
+        description: '{{`{{ $labels.configmap }} configmap is not expected.`}}'
+        opsrecipe: atlas-app-configuration/
+      expr: |
+        kube_configmap_info{cluster_type="management_cluster", configmap=~"loki-user-values|mimir-user-values"} > 0
```

- refactoring to shorten CM name when we add more app names
- don't we have some user-values named `[clusternaem]-[app]-user-values`?
```suggestion
        kube_configmap_info{cluster_type="management_cluster", configmap=~".*(loki|mimir)-user-values"} > 0
```